go-controller/pkg/ovn/gress_policy.go

```diff
@@ -246,9 +249,11 @@ func (gp *gressPolicy) isEmpty() bool {
 // by the parent NetworkPolicy)
 // buildLocalPodACLs is safe for concurrent use, since it only uses gressPolicy fields that don't change
 // since creation, or are safe for concurrent use like peerVXAddressSets
+// Note that portGroupName is portGroup's name without network scope prefix
```

Comment:
I am suggesting otherwise, to not rely on `NetworkControllerNetInfo`. 

Another option is to provide the network name and have a `GetNetworkScopedName` free function.